src/AtomsGaffer/AtomsCrowdGenerator.cpp

```diff
@@ -916,6 +933,19 @@ ConstObjectPtr AtomsCrowdGenerator::computeBranchObject( const ScenePath &parent
         return variationsPlug()->objectPlug()->getValue();
     }
 
+
+    Imath::M44f transformMtx;
+    ScenePath transfromPath;
+    if (branchPath.size() > 2) {
+        transfromPath.push_back(branchPath[1]);
+        transfromPath.push_back(branchPath[2]);
+        for (int ii = 4; ii < branchPath.size(); ++ii) {
+            transfromPath.push_back(branchPath[ii]);
+        }
+        transformMtx = variationsPlug()->fullTransform(transfromPath);
```

Comment:
Since now the atoms file contain the full heirarchy, the meshes are stored in local space. During the skinning the points are converted in world space and after the deformation they are converted back to the local space. I have access to the world matrix directly inside the atoms mesh but it desn't contain changed made by the user, so I need to take the world matrix from gaffer

Reply:
Ah, ok thanks for clarifying. Do you think the actual deformation can occur on local space in the future? Should we start a Jira issue for that?